### DISCUSSION TITLE
v3.5.1-R2.1 - Offline PC

### DISCUSSION THREAD

**Discussion by alexondrums** [intel/openvino-plugins-ai-audacity#169] [Q&A]
From v3.5.0-R2 for the Windows Installer - it says 'The installer will take care of downloading the required model files and installing them into the right system location.'

I'd like to install v3.5.1-R2.1 on my Windows 10 PC, which is **offline** - how can this be used using the Installer?

**Comment by RyanMetcalfeInt8** [comment]
Ultimately, you need to populate an ```openvino-models``` folder with the models that you are interested in, and transfer these over to your **offline** PC.

You can do this by downloading & running the installer on a PC that is connected to the internet, and in this case select all of the models that you want to download. After installation is complete, go to ```C:\Program Files\Audacity``` and zip up ```openvino-models``` folder.

Transfer the installer, and the ```openvino-models.zip``` that you created to your offline PC. 

Run the installer, and choose 'Install no models' during installation so that it doesn't try to download anything. 

Extract ```openvino-models.zip``` to the ```C:\Program Files\Audacity``` folder on your offline PC.

With this, you should be able to run.

**Reply by Etchegando59** [reply]
Devia haver outra maneira ,este instalador esta dando problemas ,tinha que ser arquivos zipados.

**Comment by MarcoRavich** [comment]
> I'd like to install v3.5.1-R2.1 on my Windows 10 PC
>
Why so outdated version ?
